01_fib/jimbotech/README.md

```diff
@@ -0,0 +1,15 @@
+# Fibonacci Number Generator
+## Overview
+As part of the ANZ "go" course the first lab is to produce a piece of software that will produce the Fibonacci series for fib(7). It was also an exercise in the tour.golang.org.
```

Comment:
nitpick: Markdown is an extension of plain old ascii documents, in that ascii markup is added so it can be prettier when rendered as markdown. But the files should still be readable as ascii documents. This means wrapping text around 80 chars (the general convention), as people use all sorts of different width terminals, and some that require side scrolling to read wider text. So please make markdown text readable as plain ascii as originally intended.

Plenty has been written about line length / text width - here's one example: https://baymard.com/blog/line-length-readability

Reply:
Done